main.go

```diff
@@ -95,11 +95,11 @@ func main() { // nolint gocyclo
 		headers := make(map[string]string)
 		if len(fheaders) > 0 {
 			for _, v := range fheaders {
-				result := strings.SplitN(v, ":", 2)
+				result := strings.SplitN(v, ": ", 2)
```

Comment:
Do you really need to include this white space here? Seems just your second changing below should enough. So, it allows people to use with or without whitespace prefix in the header value.

Reply:
Second should be enough indeed even if the first match more the key/value aspect of the thing. Will change for the 2nd only.